templates/aws-tap-entrypoint-existing-vpc.template.yaml

```
@@ -1096,7 +2133,7 @@ Resources:
             ]
           }
         - IamOidcProviderArn: !Sub arn:aws:iam::${AWS::AccountId}:oidc-provider
```

```suggestion
        - IamOidcProviderArn: !Sub arn:${AWS::Partition}:iam::${AWS::AccountId}:oidc-provider
```

Missed this one previously